packages/discovery-react-components/src/utils/document/transformEnrichment.ts

```diff
@@ -6,19 +6,34 @@ import { ENRICHMENTS, getEnrichmentName } from 'components/CIDocument/utils/enri
 import {
   EnrichedHtml,
   Contract,
+  Invoice,
+  PurchaseOrder,
   Attributes,
   Relations,
   Metadata
 } from 'components/CIDocument/types';
 import { Ontology } from './ontology/types';
 
-const modelMapping = {
-  contract: setMetadata,
-  invoice: setAttributesAndRelations,
-  purchase_order: setAttributesAndRelations
+const ontologyModelMapping = {
+  invoice: invoiceOntology,
+  purchase_order: purchaseOrderOntology
```

Comment:
It seems that our ontology mapping is missing one of our ontologies ("contract"). From a maintainability standpoint, we should try to do as much of the same thing across all of our ontologies as we can. We should special case `contracts`.

Reply:
Because `contract` does not have `ontology`. We introduced ontology for mapping attributes and relations.

Reply:
Well, perhaps we're using different meanings!

What I mean to say is that we handle 3 different types of docs. We should strive to make our code consistent in how these 3 are treated (even if 2 of them are quite close). That's what I was suggesting with my comment below about changing the `getUpdatedEnrichment` function to work with this mapping.